jetty-proxy/src/test/java/org/eclipse/jetty/proxy/AsyncMiddleManServletTest.java

```diff
@@ -159,9 +159,18 @@ private void startClient() throws Exception
     @AfterEach
     public void dispose() throws Exception
     {
-        client.stop();
-        proxy.stop();
-        server.stop();
+        if (client != null)
+            client.stop();
+        if (proxy != null)
+            proxy.stop();
+        if (server != null)
+            server.stop();
```

Comment:
```suggestion
        LifeCycle.stop(client);
        LifeCycle.stop(proxy);
        LifeCycle.stop(server);
```

Reply:
@gregw 👍 Thank you for the review. Done.